caffe2/python/pybind_state.cc

```diff
@@ -1749,7 +1749,6 @@ void addGlobalMethods(py::module& m) {
   m.def(
       "onnxifi",
       [](const py::bytes& pred_net_str,
-         const py::bytes& shapes_str,
```

Comment:
Why are you deleting this? And this would cause the code compilation failures.

Reply:
@malfet I've made the necessary changes.